app/controllers/spree/user_sessions_controller.rb

```diff
@@ -45,6 +45,7 @@ def destroy
       # Logout will clear session data including shopfront_redirect
       #   Here we store it before actually logging out so that the redirect works correctly
       @shopfront_redirect = session[:shopfront_redirect]
+      flash.notice = t('devise.sessions.signed_out')
```

Comment:
Devise already adds the flash message, we shouldn't be adding one here.

Can we drop this line?